### PR TITLE
Fix gnulib/crypto/md4 dependency

### PR DESCRIPTION
**Pull request by papajulio**
This makes SOGo compile again with GNUTLS option enabled, it is failing without this after latest changes in this commit e5d01428ad074d45ed8719b3aaac5d09537faba4

### CODE REVIEW

File: SoObjects/SOGo/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+config.h
+md4.c
```
Comment (extrafu) [inline SoObjects/SOGo/.gitignore]: Why this? remove!

File: configure [hunk 2]
```diff
@@ -374,6 +374,41 @@ EOF
...
+checkGnulib() {
+  if test ! -d $gnulib_dir; then
+    echo "When using --with-ssl=gnutls option gnulib package is required to be installed in $gnulib_dir"
+    exit 1
+  fi
+  cfgwrite "GNULIB_INCLUDE := $gnulib_dir"
+}
+
+gnulibMd4() {
+  local source_files="md4.c"
```
Comment (extrafu) [inline configure]: Where does that come from?

**Comment by papajulio** [comment]
Hey @extrafu anything that you think that will work better for the project please do it, this was only our quick solution as we need to be working fast. Please feel free to close this and do it in another way that fit you best. We only want to call your attention about the problem with the compilation.

**Comment by extrafu** [comment]
Properly fixed with https://github.com/inverse-inc/sogo/commit/beb666bc5f995e81c3cf56925038a780ad80e7f4

Please also update PR78.

**Pull request closed without merging**